pages/signup.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>SignUp</title>
+    <link rel="stylesheet" href="/signup.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="content-box">
+        <div class="header-logo">
+          <a href="/">
+            <img
+              class="loginLogoImg"
+              src="/images/loginPageLogo.png"
+              alt="판다마켓"
+            />
+          </a>
+        </div>
+
+        <div class="input-form-container">
+          <div class="input-box-container">
+            <label for="email">이메일</label>
+            <input id="email" type="email" class="input-box" />
+          </div>
+
+          <div class="input-box-container">
+            <label for="name">닉네임</label>
+            <input id="name" type="text" class="input-box" />
+          </div>
+
+          <div class="input-box-container">
+            <label for="password">비밀번호</label>
+            <input id="password" type="password" class="input-box" />
+          </div>
+
+          <div class="input-box-container">
+            <label for="conFirmPassword">비밀번호확인</label>
+            <input id="conFirmPassword" type="password" class="input-box" />
+          </div>
+        </div>
+
+        <div class="button-container">
+          <button class="signup-button">회원가입</button>
```

Comment:
💊 제안
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
          <button type="submit" class="signup-button">회원가입</button>
```

+type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.
자세한 것은 추후 학습 하시면서 배우실 것이니 우선은 button 태그의 type 속성을 명시적으로 작성하는 것이 좋다는 것만 기억해주시면 좋겠습니다~

Reply:
❗️ 수정요청
위에서 언급드린 button 동작의 이점을 가지고 가기 위해서는 form 태그를 사용하시는 것이 좋습니다!
회원가입에서 form 영역은 회원가입할때 필요한 데이터들의 구역을 의미합니다.
간편 로그인하기 부분은 회원가입시 필요한 정보가 아니니 form 태그 밖에 두시면 됩니다~
이를 참고해서 form 태그를 추가해보세요!